benchmark.py

```diff
@@ -226,23 +256,46 @@ def main():
   parser.add_argument("--sleep_between_retry",
                       help="number of seconds to sleep between each retry.",
                       type=int, default=5)
-
-  create_del_parser = parser.add_mutually_exclusive_group(required=False)
-  create_del_parser.add_argument("--create_delete", dest="create_delete",
-                                 action="store_true",
-                                 help="if you want to create/delete new VM.")
-  create_del_parser.add_argument("--no-create_delete", dest="create_delete",
-                                 action="store_false")
+  parser.add_argument("--db_instance_name",
+                      help="the name of the gcloud instance",
+                      default="envoy-db-instance")
+  parser.add_argument("--envoy_hash",
+                      help="the hash of envoy version",
+                      default="xxxxxx")
+  parser.add_argument("--runid",
+                      help="the run id of this benchmark",
+                      default="0")
+  parser.add_argument("--ownip", help=("the machine's IP from where"
+                                       " the script is being run"),
+                      default="127.0.0.1")
```

Comment:
This IP based authentication seems weird. Is there no better way to do credential based authentication to Cloud SQL? If not, you should be able to automatically infer this `--ownip` in any case.